ios/RNBranchConfig.m

```diff
@@ -20,7 +20,7 @@
 NSString * _Nonnull const RNBranchConfigDeferInitializationForJSLoadOption = @"deferInitializationForJSLoad";
 NSString * _Nonnull const RNBranchConfigEnableFacebookLinkCheck = @"enableFacebookLinkCheck";
 
-NSString * const RNBNC_PLUGIN_VERSION = @"4.3.0";
+NSString * const RNBNC_PLUGIN_VERSION = RNBRANCH_VERSION;
```

Comment:
@jdee how can I test this? I tried checking out your branch, then form inside `testbed_simple`

- npm uninstall --save react-native-branch
- npm install --save /Users/benas.klastaitis/AndroidStudioProjects/plugins/react-native-branch-deep-linking-attribution/
- open testbed_simple in XCode and build but I get the following error.

/Users/benas.klastaitis/AndroidStudioProjects/plugins/react-native-branch-deep-linking-attribution/examples/testbed_simple/node_modules/react-native-branch/ios/RNBranchConfig.m:23:41: error: use of undeclared identifier 'RNBRANCH_VERSION'
NSString * const RNBNC_PLUGIN_VERSION = RNBRANCH_VERSION;

What am I missing?

Reply:
@bklastaitis-branch You'll need to run `pod update`:

```bash
cd ios
bundle check || bundle install
bundle exec pod update
```

That will regenerate the Pods project to pass that definition through.

Reply:
TY!